addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -66,7 +66,7 @@
 
                         <field name="state" readonly="1" widget="statusbar"/>
                     </header>
-                    <div class="alert alert-info text-center" role="alert" attrs="{'invisible': ['&amp;','&amp;','&amp;',('sent', '=', 0),('ignored', '=', 0),('scheduled', '=', 0),('failed', '=', 0)]}">
+                    <div class="alert alert-info text-center mb-0" role="alert" attrs="{'invisible': ['&amp;','&amp;','&amp;',('sent', '=', 0),('ignored', '=', 0),('scheduled', '=', 0),('failed', '=', 0)]}">
```

Comment:
Doesn't it break layout if the under div is not displayed ? Actually we could merge this div and the following one and update the invisible condition accordingly. That way with only one alert-info box, no need to mess with margins.

Reply:
I was actually testing this one and it does indeed change the layout in mobile a little bit:

Before:
![Selection_700](https://user-images.githubusercontent.com/41282536/85139119-44cc9580-b244-11ea-9f05-ebd9c4b734a6.png)

After:
![Selection_701](https://user-images.githubusercontent.com/41282536/85139140-4ac27680-b244-11ea-84cf-7ae8e60550a0.png)

Reply:
Well from my perspective, it look better without margin.

> Actually we could merge this div and the following one and update the invisible condition accordingly. That way with only one alert-info box, no need to mess with margins.

Why not both ? Kill the margin and merge the alert box ?

Reply:
Well I would keep the margin, as I know I am really bad at UX and would let other people decide if this margin is ugly or not. If you really feel it is better without it you can check with LNA what how he feels about it.